attributes/network/cddl/1002_idstore.cddl

```diff
@@ -3,42 +3,51 @@ word = tstr
 idstore-recall-phrase = [2*5 word]
 idstore-address = non-anonymous-identity
 idstore-credid = bstr .size (16..1023)
+idstore-pubkey = bstr .cbor COSE_Key
 ; end::types[]
 
 ; tag::store[]
 idstore.store@param = {
-    ; The address associated to the recall phrase
+    ; The address associated with the credential ID and public key
     0 => idstore-address
 
-    ; The credential ID associated to the recall phrase
+    ; The credential ID associated with the recall phrase
     1 => idstore-credid
+
+    ; The public key associated with the recall phrase
+    2 => idstore-pubkey
 }
 idstore.store@return = {
-    ; Recall phrase associated to the address and credential ID
-    0 => idstore-recall-phrase,
+    ; The recall phrase associated with the credential ID and public key
+    0 => idstore-recall-phrase 
```

Comment:
Nit: we keep `,` at the end so that diffs on new fields don't change lines they shouldn't.